pkg/kubectl/cmd/get.go

```diff
@@ -296,26 +292,24 @@ func RunGet(f *cmdutil.Factory, out io.Writer, errOut io.Writer, cmd *cobra.Comm
 			allErrs = append(allErrs, err)
 		}
 
-		// the outermost object will be converted to the output-version, but inner
-		// objects can use their mappings
-		version, err := cmdutil.OutputVersion(cmd, clientConfig.GroupVersion)
-		if err != nil {
-			return err
-		}
 		res := ""
 		if len(infos) > 0 {
 			res = infos[0].ResourceMapping().Resource
 		}
 
-		obj, err := resource.AsVersionedObject(infos, !singular, version, f.JSONEncoder())
-		if err != nil {
-			return err
+		for ix := range infos {
+			if isFiltered, err := filter.Filter(infos[ix].Object); !isFiltered {
+				if err != nil {
```

Comment:
If `isFiltered == true` the error returned is simply "error: no filter type for <resource_type>", while not something to output here, I could glog it